beam-migrate/Database/Beam/Migrate/Types/CheckedEntities.hs

```diff
@@ -78,6 +83,10 @@ renameCheckedEntity renamer =
 unCheckDatabase :: forall be db. Database be db => CheckedDatabaseSettings be db -> DatabaseSettings be db
 unCheckDatabase db = runIdentity $ zipTables (Proxy @be) (\(CheckedDatabaseEntity x _) _ -> pure $ DatabaseEntity (unCheck x)) db db
 
+unCheckedDbLens :: forall be db. Database be db => Lens' (CheckedDatabaseSettings be db) (DatabaseSettings be db)
```

Comment:
This lets you take a checked database, make changes to the fields using standard lens syntax for unchecked databases, and have it re-assembled for you. Very useful!